codepipeline-tf/modules/xc3_codepipeline/providers.tf

```diff
@@ -0,0 +1,9 @@
+terraform {
+  required_version = ">= 1.0"

```

Comment:
I see in version are different in each provider file. make sure to keep same version in each provider file.

Reply:
updated to 1.7, the same as the other providers have.